tensorflow_addons/metrics/multilabel_confusion_matrix.py

```diff
@@ -24,6 +24,49 @@
 
 
 class MultiLabelConfusionMatrix(Metric):
+    """Computes Multi-label confusion matrix.
+
+    Class-wise confusion matrix is computed for the
+    evaluation of classification.
+
+    If multi-class input is provided, it will be treated
+    as multilabel data.
+
+    Consider classification problem with two classes.
+    Resultant matrix `M` will be in the shape of (num_classes, 2, 2).
+
+    Every class has a dedicated 2*2 matrix that contains:
+    - true negatives for class i in M(0,0)
```

Comment:
`i` has not been defined.

Reply:
I have modified the comment. Can you please see if it is ok and the user can understand it?

Reply:
Yup, looks good.